content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -29,14 +29,19 @@ The following methods exist for installing kubectl on macOS:
 
 1. Download the latest release:
 
-   ```bash
+   {{< tabs name="download_binary_macos" >}}
+   {{< tab name="Intel" codelang="bash" >}}
    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/amd64/kubectl"
-   ```
+   {{< /tab >}}
+   {{< tab name="Apple Silicon" codelang="bash" >}}
+   curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/arm64/kubectl"
+   {{< /tab >}}
+   {{< /tabs >}}
 
    {{< note >}}
    To download a specific version, replace the `$(curl -L -s https://dl.k8s.io/release/stable.txt)` portion of the command with the specific version.
 
-   For example, to download version {{< param "fullversion" >}} on macOS, type:
+   For example, to download version {{< param "fullversion" >}} on Intel macOS, type:
```

Comment:
Should we give the example for apple silicone as well?

Reply:
It's possible, I didn't because in the rendered page it looked like it was a single example. I could copy the tab box and have it render the same tabs under this example.

Reply:
Could you please do that as well?

Reply:
I've done that - tabs seemed to render weird in the infobox so I went with a different way to add the examples.